analyses/tp53_nf1_score/05-evaluate-classifier.py

```diff
@@ -65,45 +65,33 @@
 np.random.seed(123)
 
 # read TP53/NF1 alterations
-status_df = pd.read_table(status_file, low_memory=False)
+full_status_df = pd.read_table(status_file, low_memory=False)
 # read in clinical file
 clinical_df = pd.read_table(clinical)
 # select only IDs
 clinical_df = clinical_df[
     ["Kids_First_Biospecimen_ID", "sample_id", "Kids_First_Participant_ID"]
 ]
 
-
-# add clinical info to alterations dataframe
-status_df = status_df.merge(
-    clinical_df,
-    how="left",
-    left_on="Tumor_Sample_Barcode",
-    right_on="Kids_First_Biospecimen_ID",
-)
-
-# Value count of variant classification
-print(status_df.Variant_Classification.value_counts())
-
-
 # Obtain a binary status matrix
-full_status_df = pd.crosstab(
-    status_df["sample_id"], status_df.Hugo_Symbol, dropna=False
-)
-full_status_df.head(3)
-full_status_df[full_status_df > 1] = 1
-full_status_df = full_status_df.reset_index()
-full_status_df = full_status_df.drop(["No_TP53_NF1_alt"], axis=1)
+full_status_df['tp53_status']= full_status_df.apply(
+    lambda x: (1
+               # IF
+               # both SNV/CNV counts are more than 1 so can be considered as a TP53 double hit
+               if (x['SNV_indel_counts'] !=0 and x['CNV_loss_counts'] != 0 or
```

Comment:
I am not sure I understand this line:  the line above says "more than 1" but do you mean "1 or more"? If so, I would probably change these from `!= 0` to `>= 1` just for clarity.

The other minor point is that I tend to find this `ifelse` structure confusing if it spans more than one line. I would use the more conventional:
if( condition1 or
    condition2):
  1 
else:
  0

Reply:
updated the code with this suggestion, it does look more readable. Thanks!